bin/deploy

```diff
@@ -112,10 +110,23 @@ test -f "$LSSTSW/miniconda/.packages.deployed" || ( # conda packages
     fi
 
     if [[ $BLEED_DEPLOY == true ]]; then
-        # The conda Intel MKL linked packages are intentionally avoided.
-        # See: https://jira.lsstcorp.org/browse/DM-5105
-        CONDA_BLEED_PACKAGES=(nomkl numpy scipy matplotlib requests cython sqlalchemy astropy pandas future)
+        # lsst_distrib / lsst_sims deps
+        CONDA_BLEED_PACKAGES=(numpy scipy matplotlib requests cython sqlalchemy astropy pandas future)
         ARGS+=("${CONDA_BLEED_PACKAGES[@]}")
+
+        # matplotlib pulls in qt and there are concerns about qt5 causing
+        # breakage, so we are pinning to 4.x
+        ARGS+=("pyqt==4.11.4")
```

Comment:
I want bleed to be bleed. When you are coming up with a new package list then forcing it in there temporarily is fine with me. I just don't want bleed to not be bleed just because we are waiting on Continuum for this particular problem to be fixed.